tools/semgrep-py.yml

```diff
@@ -16,6 +16,8 @@ rules:
     paths:
       include:
         - zerver/views/
+      exclude:
+        - zerver/views/channel_folders.py
```

Comment:
Rather than adding this exclude, I think we should just have the `check_folder_in_use` function live in `zerver/lib/channel_folders.py`.

Reply:
Fixed.